### DISCUSSION TITLE
Docker log

### DISCUSSION THREAD

**Discussion by SaasX** [postalserver/postal#2699] [Help with using Postal]
Hello, 

I use little VPS for my customer who want a simple MTA, and for some, I have disk space problems.

to limit costs and not take a larger server, i set up a cron job on a test server to delete Docker log files every day.

Do you think this could cause message loss?

**Comment by willpower232** [comment]
That should be fine but actually you should have a read of this https://github.com/postalserver/postal/discussions/2544#discussioncomment-6626271

You can configure docker log size yourself and then you don't need a manual cron job to tidy it up.
